src/runtime-rs/src/shim/src/bin/containerd-shim-kata-v2.rs

```diff
@@ -15,10 +17,11 @@ enum Action {
     Help,
 }
 
-fn parse_args() -> Action {
+fn parse_args(args: &[OsString]) -> Action {
```

Comment:
Again, all this re-organization makes review difficult.  Please rebase to make a *logical* separation between each patch.

Reply:
Fixed in the second revision: rebased the patches by logical separations